float32[24] ranges
float32 obstacle_min_range
float32 obstacle_angle
float32 heading
float32 current_distance
bool done
